components/adblock_rust_ffi/src/wrapper.h

```diff
@@ -56,6 +56,7 @@ typedef ADBLOCK_EXPORT struct FilterListMetadata {
 
```

Comment:
FilterListMetadata has a default ctor.
What expires should be set in that case?

Reply:
Good catch - I added a default value in `SubscriptionInfo` but this should get one too.

Reply:
added default value